rascal2/ui/view.py

```diff
@@ -92,6 +94,15 @@ def setupMDI(self):
             "Fitting Controls": self.controls_widget,
         }
 
+        project_path = pathlib.Path(self.presenter.model.save_path)
```

Comment:
Maybe the logging and setting init should be in its own method i.e. `initialize_setting_and_log` then called in createProject so this function just manages mdi layout.